### PR TITLE
SJSU-1034 Confirmation Page

### PR DESCRIPTION
**Pull request by minhphan156**
Description:
- implemented new confirmation page
- created and connected with dummy data in bookingReducer
- implemented mobile friendly version of the page
  
How to test:
- pull from SJSU-1034 and resize the window to smaller size

### CODE REVIEW

**Comment by SandroSal** [comment]
- decimal for nightly rate
- please add padding to left of "RESERVATION DETAILS" and "Policies"
- either have both of them all caps, or not
- additional stuff you could list: # of rooms, # of nights, taxes & fees

overall looks really good! 👍

File: front_end/src/components/booking/Confirmation.js [hunk 1]
```diff
@@ -1,50 +1,176 @@
...
+      </Grid>
+
+      <Grid item xs={isWidthDown("xs", width) ? 11 : 4}>
+        <div
+          style={{
+            color: "white",
+            background: "linear-gradient(to right, #0c4b78, #3d4e96, #2c76a9)"
+          }}
+        >
+          {" "}
```
Comment (SandroSal) [inline front_end/src/components/booking/Confirmation.js]: there is a couple {" "}, are they needed?

File: front_end/src/components/booking/Confirmation.js [hunk 2]
```diff
@@ -1,50 +1,176 @@
...
+          style={{ marginTop: 40 }}
+        >
+          {bookingData.hotelName}
+        </Typography>
+        <Typography variant="h6" gutterBottom align="center">
+          {bookingData.hotelAddress}
+        </Typography>
+      </Grid>
+
+      <Grid item xs={isWidthDown("xs", width) ? 11 : 5}>
```
Comment (SandroSal) [inline front_end/src/components/booking/Confirmation.js]: everybody using 'isWidthDown' now 😆

File: front_end/src/components/booking/BookingDetailsCard.js [hunk 3]
```diff
@@ -8,6 +8,7 @@ import Button from "@material-ui/core/Button";
 import Typography from "@material-ui/core/Typography";
 import lightBlue from "@material-ui/core/colors/lightBlue";
 import SvgIcon from "@material-ui/core/SvgIcon";
+import { connect } from "react-redux";
```
Comment (SandroSal) [inline front_end/src/components/booking/BookingDetailsCard.js]: are you even using this file?

Comment (SandroSal) [reply front_end/src/components/booking/BookingDetailsCard.js]: I'm not referring to line 11, but the file as a whole

File: front_end/src/components/booking/Confirmation.js (hunk 1 above)
Comment (minhphan156) [reply front_end/src/components/booking/Confirmation.js]: removed

File: front_end/src/components/booking/Confirmation.js (hunk 2 above)
Comment (minhphan156) [reply front_end/src/components/booking/Confirmation.js]: credited to @DaleCS

File: front_end/src/components/booking/BookingDetailsCard.js (hunk 3 above)
Comment (minhphan156) [reply front_end/src/components/booking/BookingDetailsCard.js]: removed the files

File: front_end/src/components/booking/Confirmation.js [hunk 4]
```diff
@@ -1,50 +1,237 @@
...
+        </Grid>
+
+        <Grid item xs={isWidthDown("xs", width) ? 11 : 5}>
+          <CardMedia
+            style={
+              isWidthDown("xs", width)
+                ? { width: "100%", height: 200, marginTop: 20 }
+                : { width: "100%", height: 200 }
+            }
+            image={require("../landing_page/SD.jpg")} // *********need to be updated with real PICTURES
```
Comment (minhphan156) [inline front_end/src/components/booking/Confirmation.js]: need to have real data to test this

File: front_end/src/components/booking/Confirmation.js [hunk 5]
```diff
@@ -1,50 +1,237 @@
...
+          </Typography>
+          <Typography variant="h6" gutterBottom align="center">
+            {bookingData.hotelAddress}
+          </Typography>
+        </Grid>
+
+        <Grid item xs={isWidthDown("xs", width) ? 11 : 5}>
+          <CardMedia
+            style={{ width: "100%", height: 200 }}
+            image={require("../landing_page/SF.jpg")} // *********need to be updated with real PICTURES
```
Comment (minhphan156) [inline front_end/src/components/booking/Confirmation.js]: need to have real data from backend to test this

**Pull request merged**